CODE_OF_CONDUCT.md

```diff
@@ -61,7 +61,7 @@ representative at an online or offline event.
 
 Instances of abusive, harassing, or otherwise unacceptable behavior may be
 reported to the community leaders responsible for enforcement at
-[INSERT CONTACT METHOD].
+[this email](contact@overture.bio).
```

Comment:
I think just show the email instead of `this email`? Not sure what style guides would recommend:

```suggestion
[contact@overture.bio](contact@overture.bio).
```